src/main/java/io/kabanero/tools/KabaneroTool.java

```diff
@@ -24,10 +24,14 @@
     public String namespace;
     public String route;
     public String location;
+    public String instancePageCardText;
+    public String instancePageButtonText;
     
-    public KabaneroTool(String name, String location){
+    public KabaneroTool(String name, String location, String instancePageCardText, String instancePageButtonText){
         this.name = name;
         this.location = location;
+        this.instancePageCardText = instancePageCardText;
+        this.instancePageButtonText = instancePageButtonText;
```

Comment:
I would just name these cardText and ActionText to keep them more generic, for example, maybe it won't be an "instance page" in the future.